webapp/model/products.json

```diff
@@ -0,0 +1,106 @@
+{  
+    "Products": [
+        {
+            "Id": "1",
+            "Name": "tea",
+            "Description": "very useful thing",
+            "Price": 2.65,
+            "Quantity": "134",
+            "Supplier": ["1", "2"],
+            "Category": "relax",
+            "Availability": true
+        },
+        {
+            "Id": "2",
+            "Name": "coffee",
+            "Description": "very cheerful thing",
+            "Price": 21.65,
+            "Quantity": "22",
+            "Supplier": ["1", "3"],
+            "Category": "enjoy",
+            "Availability": false
+        },
+        {
+            "Id": "3",
+            "Name": "computer",
+            "Description": "very informative thing",
+            "Price": 2.65,
+            "Quantity": "134",
+            "Supplier": ["4"],
+            "Category": "work",
+            "Availability": true
+        },
+        {
+            "Id": "4",
+            "Name": "car",
+            "Description": "very comfortable thing",
+            "Price": 21.65,
+            "Quantity": "22",
+            "Supplier": ["1", "3", "5"],
+            "Category": "speed",
+            "Availability": false
+        }
+    ],
+    "Categories": [
```

Comment:
I asked you to split products.json file into 3 separate ones so you could remove Categories and Suppliers entities from products.json file